boards/arm/frdm_k64f/frdm_k64f-pinctrl.dtsi

```diff
@@ -0,0 +1,130 @@
+/*
+ * NOTE: Autogenerated file by kinetis_signal2dts.py
+ *       for MK64FN1M0VLL12/signal_configuration.xml
+ *
+ * Copyright (c) 2022, NXP
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <nxp/kinetis/MK64FN1M0VLL12-pinctrl.h>
+
+&pinctrl {
+	i2c0_default: i2c0_default {
+		group0 {
+			pinmux = <I2C0_SCL_PTE24
+				I2C0_SDA_PTE25>;
```

Comment:
nit: `pinmux = <I2C0_SCL_PTE24>, <I2C0_SDA_PTE25>;` (should be equivalent, but it's the format found in the pinctrl guide, also used by STM32 in Linux Kernel).